README.md

```diff
@@ -27,23 +23,29 @@ Effects Tab provides 9 effects for on-the-fly image editing on upload:
 crop, rotate, mirror, flip, blur, sharpen, enhance, grayscale and invert.
 You can customize which effects are allowed and otherwise affect the tab behavior.
 
-* [Demo](https://uploadcare.github.io/uploadcare-widget-tab-effects/)
+<!-- toc -->
+
 * [How it works](#how-it-works)
+* [Demo](#demo)
 * [Requirements](#requirements)
 * [Install](#install)
 * [Usage](#usage)
 * [Configuration](#configuration)
 * [Options](#options)
 * [Localization](#localization)
+* [Security issues](#security-issues)
+* [Feedback](#feedback)
+
+<!-- tocstop -->
 
 ## How it works
 
-Image operations provided by Effects Tab are based on the capabilities of our [CDN API][uc-cdn].
-The tab outputs a CDN link holding your image [UUID][uc-uuid] and
-[image operations][uc-cdn-image-operations] applied by a user during the in-tab editing process.
+Image operations provided by Effects Tab are based on the capabilities of our [CDN API][uc-docs-cdn].
```

Comment:
... the capabilities of Uploadcare `[Image Processing][uc-feature-image-processing]` ...